operators/cloud-native-postgresql/1.23.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1331 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "Backup",
+          "metadata": {
+            "name": "backup-sample"
+          },
+          "spec": {
+            "cluster": {
+              "name": "cluster-sample"
+            }
+          }
+        },
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "Cluster",
+          "metadata": {
+            "name": "cluster-sample"
+          },
+          "spec": {
+            "instances": 3,
+            "logLevel": "info",
+            "primaryUpdateStrategy": "unsupervised",
+            "storage": {
+              "size": "1Gi"
+            },
+            "walStorage": {
+              "size": "1Gi"
+            }
+          }
+        },
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "Pooler",
+          "metadata": {
+            "name": "pooler-sample-rw"
+          },
+          "spec": {
+            "cluster": {
+              "name": "cluster-sample"
+            },
+            "instances": 1,
+            "pgbouncer": {
+              "poolMode": "session"
+            },
+            "type": "rw"
+          }
+        },
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "ScheduledBackup",
+          "metadata": {
+            "name": "scheduledbackup-sample"
+          },
+          "spec": {
+            "cluster": {
+              "name": "cluster-sample"
+            },
+            "schedule": "0 0 0 * * *"
+          }
+        }
+      ]
+    capabilities: Auto Pilot
+    categories: Database
+    certified: "true"
+    containerImage: quay.io/enterprisedb/cloud-native-postgresql@sha256:ba9c00390c41a6fd147ce2263cda09b89958edeed69b0610f0095da1dd1580dd
+    createdAt: "2024-04-25T11:05:52Z"
+    description: EDB Postgres for Kubernetes is an open source operator designed to manage highly available PostgreSQL databases with a primary/standby architecture on any supported Kubernetes cluster.
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    olm.skipRange: '>= 1.18.0 < 1.23.0'
```

Comment:
skips is an array and must be formatted properly. (note GH isn't formatting the below properly for whatever reason)

skips:
   - cloud-native-postgresql.v1.22.0

Reply:
@itsmitul9 wrap your code examples using the `code` Markdown wrapper - three backticks in a row.
```
skips:
  - cloud-native-postgresql.v1.22.0
```

Reply:
@jfrancin @itsmitul9 

This error is regarding different ocp versions, this probably be a common issue if we want to support one more ocp versions, did the skips could ensure olm upgrade all versions in range '>= 1.18.0 < 1.23.0' ?

```
Replaces bundle Bundle(cloud-native-postgresql/1.22.0) ['4.12', '4.13', '4.14'] does not support the same OCP versions as bundle Bundle(cloud-native-postgresql/1.23.0) ['4.12', '4.13', '4.14', '4.15']
```

Reply:
we have a similar case here https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3797

The support help solved this pr weeks ago due to same error https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3771.